pilz_robot_programming/src/pilz_robot_programming/commands.py

```diff
@@ -417,9 +437,9 @@ def __str__(self):
         out_str = BaseCmd.__str__(self)
         if self._relative:
             out_str += " relative: True"
-        if isinstance(self._goal, Pose):
+        if isinstance(self._goal, Pose) or isinstance(self._goal, PoseStamped):
             out_str += " Cartesian goal:\n" + str(self._goal)
-        if isinstance(self._goal, list):
+        elif isinstance(self._goal, tuple):
```

Comment:
This is already not sweet, and will get quite messy, with more types. We should try to find an alternative.